CHANGES.md

```diff
@@ -0,0 +1,13 @@
+# Changes
+
+## Version 2.0
+
+* Dynamic license header check - #37
```

Comment:
Does it make sense to include "everything" here such as these things, or should we keep it down to "user facing" changes?

Reply:
I think user/dev facing changes should be mentioned. I.e. not updating karaf test version, but if we change an interface.

Reply:
Sounds reasonable.

Another way to slice would be to keep user facing stuff only in the changelog, and let developers use git log. No strong opinion though.